capstone-client/src/app/app.module.ts

```diff
@@ -14,10 +14,11 @@ import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {MatInputModule} from '@angular/material/input';
 import {MatIconModule} from '@angular/material/icon';
 import {MatButtonModule} from '@angular/material/button';
-import {FormsModule} from '@angular/forms';
+import {FormsModule, ReactiveFormsModule} from '@angular/forms';
 
 // import httpClient
 import {HttpClientModule} from '@angular/common/http';
+import { ForgetComponent } from './forget/forget.component';
```

Comment:
remove this since it is not part of the current context
and reformat the code using ctrl + alt + L